src/EPR.Payment.Service/Controllers/PaymentsController.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Asp.Versioning;
```

Comment:
Please use cancellationtoken to free up the resources. I have send you the modified files. There are 7 files which I have changed but please make sure you change the tests to include cancellationtoken

![image](https://github.com/DEFRA/epr-payment-service/assets/167305073/a6c95c8e-a7b5-4db6-bf15-867bee4549ea)

Reply:
resolved